.github/workflows/build.yml

```diff
@@ -9,71 +9,125 @@ on:
       - main
 
 jobs:
-  determine-service-matrix:
+  service-matrix:
     runs-on: ubuntu-latest
     outputs:
-      services: ${{ steps.determine-matrix.outputs.changes }}
+      changes: ${{ steps.determine-matrix.outputs.changes }}
+      # IF you add a new filter it ALSO needs to be here
+      services: >-
+        ["account", "graph", "content-publishing", "content-watcher"]
```

Comment:
I tried to find a way not to have to do this, but no luck. This was still the best way.